src/components/PokeCard.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { mockPokemonData } from "../mock/pokeData.js";
+
+const PokeCard = () => {
+  let { name, sprites, video } = mockPokemonData;
```

Comment:
It would be better to send mock data to the PokeCard component

Reply:
Oh, of course. I'll add it and request your review again.
Thank you!

Reply:
> It would be better to send mock data to the PokeCard component

Added the suggested changes, plus other minimal stuff.